syntax = "proto3";
package ingress;




import "container_sdk/model/container/ingress.proto";
import "container_sdk/model/container/namespace.proto";

/*
Get请求
*/
message GetRequest {
    /*
    Ingress instance id
    */
    string instanceId = 1 ;
}

/*
Get返回
*/
message GetResponse {
    /*
    Ingress
    */
    container.Ingress ingress = 1 ;
    /*
    Namespace
    */
    container.Namespace namespace = 2 ;
}

/*
GetApi返回
*/
message GetResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    ingress.GetResponse data = 4 ;
}

